tests/e2e/tools/lib/editor.js

```diff
@@ -19,24 +21,71 @@ async function openComponentPanel({page, editor}, panelTitle) {
   return editorSettings;
 }
 
+/**
+ * Close the block inserter
+ *
+ * @param {{Page}} page
+ */
+const closeBlockInserter = async ({page}) => {
+  const closeSidebar = await page.getByRole('button', {name: 'Close block inserter'});
+  if (await closeSidebar.isVisible()) {
+    await closeSidebar.click({force: true});
+    await expect(closeSidebar).toBeHidden();
```

Comment:
```suggestion
   await page.waitForTimeout(500);
```

I am not sure why the expectation here, but in my testing, if this runs fast enough, we run into an instance where the UI has not updated so the button is still present and this assertion fails. Since we are just closing the sidebar, we can add a small delay to ensure the UI updates, then remove the assertion, seems to work well with no issues in my tests